account_invoice_comment_template/models/account_move.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2014 Guewen Baconnier (Camptocamp SA)
+# Copyright 2013-2014 Nicolas Bessi (Camptocamp SA)
+# Copyright 2018 Tecnativa - Vicent Cubells
+# Copyright 2019 Iván Todorovich (Druidoo)
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    comment_template1_id = fields.Many2one(
```

Comment:
Why do you still keep the code in here, isn't it easier to treat this in the migration script to include also partners. Like this you can remove the code, tests, views, translations...

Reply:
What do you think about it @pedrobaeza ?

Reply:
I think the approach is incorrect: we still need the specific custom texts on each document, as they can modify them.